PowerControlPanel/Classes/StartUp/Resources/StringResources.ru.xaml

```diff
@@ -2,127 +2,127 @@
 
     
-    <system:String x:Key="HomePage_TDPSettings">TDP Settings</system:String>
-    <system:String x:Key="HomePage_CPUSettings">CPU Settings</system:String>
-    <system:String x:Key="HomePage_DisplaySettings">Display Settings</system:String>
-    <system:String x:Key="HomePage_AMDSettings">AMD Settings</system:String>
-    <system:String x:Key="HomePage_ProfileSettings">Profile Settings</system:String>
-    <system:String x:Key="HomePage_VolumeSettings">Volume Settings</system:String>
-    <system:String x:Key="HomePage_BrightnessSettings">Brightness Settings</system:String>
-    <system:String x:Key="HomePage_FPSSettings">FPS Limit Settings</system:String>
+    <system:String x:Key="HomePage_TDPSettings">Настройки TDP</system:String>
+    <system:String x:Key="HomePage_CPUSettings">Настройки CPU</system:String>
+    <system:String x:Key="HomePage_DisplaySettings">Настройки дисплея</system:String>
+    <system:String x:Key="HomePage_AMDSettings">Настройки AMD</system:String>
+    <system:String x:Key="HomePage_ProfileSettings">Настройки профилей</system:String>
+    <system:String x:Key="HomePage_VolumeSettings">Настройка громкости</system:String>
+    <system:String x:Key="HomePage_BrightnessSettings">Настройка яркости</system:String>
+    <system:String x:Key="HomePage_FPSSettings">Настройка ограничения FPS</system:String>
     
     <!-- Settings {DynamicResource HomePage_Brightness} -->
 
-    <system:String x:Key="Settings_UpdateSettings">Check For Updates</system:String>
-    <system:String x:Key="Settings_GeneralSettings">GENERAL SETTINGS</system:String>
-    <system:String x:Key="Settings_AMDSettings">AMD SETTINGS</system:String>
-    <system:String x:Key="Settings_IntelSettings">INTEL SETTINGS</system:String>
-    <system:String x:Key="Settings_SaveSettings">Save Settings</system:String>
-    <system:String x:Key="Settings_Theme">Theme</system:String>
-    <system:String x:Key="Settings_StartWithWindows">Start With Windows</system:String>
-    <system:String x:Key="Settings_MinimumTDP">Minimum TDP</system:String>
-    <system:String x:Key="Settings_MaximumTDP">Maximum TDP</system:String>
-    <system:String x:Key="Settings_QuickAccessMenuWidth">Quick Access Menu Width (%)</system:String>
-    <system:String x:Key="Settings_CombineTDP">Combine Sustained and Boost TDP</system:String>
-    <system:String x:Key="Settings_MainWindowStyle">Main Window Home Page Style (Restart App)</system:String>
-    <system:String x:Key="Settings_QAMStyle">Quick Access Menu Home Page Style</system:String>
-    <system:String x:Key="Settings_Language">Language/语言</system:String>
+    <system:String x:Key="Settings_UpdateSettings">Проверить наличие обновлений</system:String>
+    <system:String x:Key="Settings_GeneralSettings">Общие настройки</system:String>
+    <system:String x:Key="Settings_AMDSettings">Настрокйи AMD</system:String>
+    <system:String x:Key="Settings_IntelSettings">Настройки INTEL</system:String>
```

Comment:
Why do we dropping CAPS LOCK here? Minor: probably should it match between translations...

Reply:
Agree